content/docs/introducing-jsx.md

```diff
@@ -6,31 +6,35 @@ prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Kijk eens naar de volgende variabele toewijzing:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Deze rare tag syntax is noch een string, noch HTML.
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Dit heet JSX en het is een syntax uitbreiding voor JavaScript.
```

Comment:
```suggestion
Dit heet JSX en is een syntax aanvulling op JavaScript.
```
I think extension in this sentence can be better translated with "aanvulling" instead of "uitbreiding". Also solves the preposition issue :).